src/uwtools/tests/drivers/test_forecast.py

```diff
@@ -30,37 +30,32 @@ def test_schema():
 
 def test_create_config(tmp_path):
     """
-    Test that providing a yaml base input file and a config file will create and update yaml config
+    Test that providing a YAML base input file and a config file will create and update YAML config
     file.
     """
 
     config_file = fixture_path("fruit_config_similar.yaml")
     input_file = fixture_path("fruit_config.yaml")
     output_file = (tmp_path / "test_config_from_yaml.yaml").as_posix()
-
-    forecast_obj = FV3Forecast()
-    forecast_obj.create_model_config(
-        base_file=input_file, config_file=config_file, outconfig_file=output_file
-    )
-
+    with patch.object(FV3Forecast, "_validate", return_value=True):
+        forecast_obj = FV3Forecast(config_file=config_file)
+    forecast_obj._create_model_config(base_file=input_file, outconfig_file=output_file)
     expected = config.YAMLConfig(input_file)
     expected.update_values(config.YAMLConfig(config_file))
     expected_file = tmp_path / "expected_yaml.yaml"
     expected.dump_file(expected_file)
-
     assert compare_files(expected_file, output_file)
 
 
-def test_create_directory_structure(tmp_path):
+def test__create_directory_structure(tmp_path):
     """
-    Tests create_directory_structure method given a directory.
+    Tests _create_directory_structure method given a directory.
     """
 
     rundir = tmp_path / "rundir"
-    forecast_obj = FV3Forecast()
 
     # Test delete behavior when run directory does not exist.
-    forecast_obj.create_directory_structure(rundir, "delete")
+    FV3Forecast.create_directory_structure(rundir, "delete")
```

Comment:
Instead of creating a no-config `FV3Forecast` object, simply test the now-static method.